lib/wallet/rpc.js

```diff
@@ -1912,18 +1925,22 @@ class RPC extends RPCBase {
     if (bid > value)
       throw new RPCError(errs.TYPE_ERROR, 'Invalid bid.');
 
-    const tx = await wallet.sendBid(name, bid, value);
+    const tx = await wallet.sendBid(name, bid, value, { account });
 
     return tx.getJSON(this.network);
   }
 
   async sendReveal(args, help) {
-    if (help || args.length > 1)
-      throw new RPCError(errs.MISC_ERROR, 'sendreveal "name"');
+    if (help || args.length > 2)
+      throw new RPCError(errs.MISC_ERROR, 'sendreveal "name" ( account )');
```

Comment:
Looks like this method has a `sendRevealAll` fallback if `name` is missing. That will be hard to do since SOME name (param#0) will be required to make `account` param#1. I wonder if we should also add a `*` wildcard to trigger the `sendRevealAll` the same way `!name` would. Then you could still also pass an account.

Reply:
Is your thinking like..
# Current
`hsw-rpc sendreveal` # Triggers wallet to send reveal`
`hsw-rpc sendreveal namestring` # Triggers wallet to sendreveal using from default account
`hsw-rpc sendreveal namestring accountname` # Triggers wallet to sendreveal using addresses from accountname
# Proposed
`hsw-rpc sendreveal` * # Triggers wallet to sendrevealall using default account
`hsw-rpc sendreveal` # Missing wild-card/name/accountname throws error?
`hsw-rpc sendreveal * accountname` # Triggers wallet to sendrevealall using addresses from accountname?

Reply:
Um more like, I think
### current:
```
# sends reveal for ALL names with wallet.sendRevealAll()
hsw-rpc sendreveal

# sends reveal for one specific name with wallet.sendReveal()
hsw-rpc sendreveal <name>
```

### proposed
```
# sends reveal for ALL names with wallet.sendRevealAll() from default account
hsw-rpc sendreveal

# sends reveal for ALL names with wallet.sendRevealAll() from specific account
hsw-rpc sendreveal * <account>

# sends reveal for one specific name with wallet.sendReveal() from default account
hsw-rpc sendreveal <name>

# sends reveal for one specific name with wallet.sendReveal() from specific account
hsw-rpc sendreveal <name> <account>
```

I'm suggesting using `*` to trigger `sendRevealAll()` but I don't l know if that fits with other rpc conventions... just a thought

Reply:
...yeah... I dunno. Looks like account-specificity wasn't part of the design here and adding it in is either gunna be a major refactor or a little hacky. You could maybe forget the option of "revealing all" from a single account (for now?) Because param #0 (name) will always be required for you to also have a param #1 (account) you could just forget the whole `*` idea, and that would just mean if you specify an account, you ALSO have to specify a SINGLE name

Reply:
That seems like its gonna be the plan :)